tests/test_authz_Authorizer.py

```diff
@@ -143,7 +188,9 @@ def test_validate_raise_server_error(self):
             self.authz.validate("function_no_name")
```

Comment:
Why does it raise server error ? Could you please indicate the cause of that error somewhere?